### ISSUE TITLE
Investigate adding Digital Atlas catalogue

### ISSUE THREAD

**Issue by nf-s** [TerriaJS/nationalmap#1255]
Powered by ArgGis Hub

- https://digital.atlas.gov.au//api/search/v1/catalog
- https://digital.atlas.gov.au/api/search/v1/collections/dataset/items

**Comment by dbutrovski** [comment]
This work needs confirmation from the Digital Atlas product owner.

**Comment by nf-s** [comment]
Of course. This ticket mainly serves for my own reference, as we are working on ArcGis Hub/OGC API Records support for other Terria customers.

We won't be making code or catalogue changes to NationalMap without confirmation from Digital Atlas product owner and yourself
